### PR TITLE
Move styles and footer to Next app

### PR DESCRIPTION
**Pull request by clockblocker**

### CODE REVIEW

**Comment by sergous** [comment]
@clockblocker 
Are we really need to move old code "as-is" in one PR and adapt it later? 
How about split it to several PR and merge it slowly, part by part?

1. Wait until https://github.com/dtpstat/website/pull/15 will be merged for use styled components theme instead of global css files @kachkaev 
2. Replace redux with hooks with provider, using existing code as example
3. Create basic layout components with no styling and implementation
3. Migrate every component one by one, replace css with styled components and send as separate RR for every component

File: src/components/comment-input.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const SubmitButtonContainer = styled.div`
 `;
 
 export const CommentInput: React.VoidFunctionComponent = () => {
+  const { user } = useUser();
```
Comment (kachkaev) [inline src/components/comment-input.tsx:36]:
```suggestion
```

**Comment by kachkaev** [comment]
https://deploy-preview-14--dtp-stat-on-nextjs.netlify.app/iframes/map 👀

File: src/components/inherited-map/inherited-map.tsx [hunk 2]
```diff
@@ -1,19 +1,19 @@
-import React from "react";
+/* eslint-disable spaced-comment */
```
Comment (kachkaev) [inline src/components/inherited-map/inherited-map.tsx:1]: Maintains file diff on renaming

File: .eslintrc.cjs [hunk 3]
```diff
@@ -7,4 +7,39 @@ module.exports = {
...
+        "import/no-default-export": "off",
+        "no-param-reassign": "off",
+        "unicorn/consistent-destructuring": "off",
+        "unicorn/consistent-function-scoping": "off",
+        "unicorn/no-array-callback-reference": "off",
+        "unicorn/no-array-for-each": "off",
+        "unicorn/no-null": "off",
+      },
+    },
+  ],
```
Comment (kachkaev) [inline .eslintrc.cjs:44]: We’ll gradually remove these in follow-up PRs

File: tsconfig.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```
Comment (kachkaev) [inline tsconfig.json:3]: Does not affect Next output, but saves us from cryptic tsc errors in `for of` loops

File: src/styles/inherited-scss/helpers/_icons.scss [hunk 5]
```diff
@@ -0,0 +1,141 @@
+@use "sass:math";
```
Comment (kachkaev) [inline src/styles/inherited-scss/helpers/_icons.scss:1]: Caused by `npx sass-migrator division **/*.scss` (was needed to silence a warning)

**Pull request closed without merging**